.gitignore

```diff
@@ -223,3 +223,5 @@ nuget.exe
 *.nuget.props
 *.nuget.targets
 *.etl
+*.xml
+*.md
```

Comment:
Are we sure we can add `*.xml` and `*.md` to .gitignore at the repo root? Wouldn't this be too broad?